plugin/src/soundPlayer.lua

```diff
@@ -0,0 +1,39 @@
+-- Sounds only play in Edit mode when parented to a plugin widget, for some reason
+local plugin = plugin or script:FindFirstAncestorWhichIsA("Plugin")
+local widget = plugin:CreateDockWidgetPluginGui("Rojo_soundPlayer", DockWidgetPluginGuiInfo.new(
+	Enum.InitialDockState.Float,
+	false, true,
+	10, 10,
+	10, 10
+))
+widget.Name = "Rojo_soundPlayer"
+widget.Title = "Rojo Sound Player"
+
+local SoundPlayer = {}
+SoundPlayer.__index = SoundPlayer
+
+function SoundPlayer.new(settngs)
+	return setmetatable({}, SoundPlayer)
+end
```

Comment:
This constructor is never used, so we can save a bit of complexity by removing this, and then assuming that we'll always have settings.